pytorch_forecasting/models/nbeats/__init__.py

```diff
@@ -1,17 +1,17 @@
 """N-Beats model for timeseries forecasting without covariates."""
 
+from pytorch_forecasting.models.nbeats._grid_callback import GridUpdateCallback
 from pytorch_forecasting.models.nbeats._nbeats import NBeats
+from pytorch_forecasting.models.nbeats._nbeats_adapter import NBeatsAdapter
 from pytorch_forecasting.models.nbeats._nbeats_pkg import NBeats_pkg
-from pytorch_forecasting.models.nbeats.sub_modules import (
-    NBEATSGenericBlock,
-    NBEATSSeasonalBlock,
-    NBEATSTrendBlock,
-)
+from pytorch_forecasting.models.nbeats._nbeatskan import NBeatsKAN
+from pytorch_forecasting.models.nbeats._nbeatskan_pkg import NBeatsKAN_pkg
 
 __all__ = [
     "NBeats",
-    "NBEATSGenericBlock",
```

Comment:
to avoid breaking existing code, we need to keep exporting these in the current location, so import them from the networks location.

We should also add a `# todo v2: remove imports` to those, at a major we can break.